Cargo.toml

```diff
@@ -1,24 +1,26 @@
 [workspace]
-resolver = "2"
 members = [
     "galileo",
     "galileo-mvt",
     "galileo-types",
 ]
+resolver = "2"
 
 exclude = [
+    "galileo/examples/with_egui",
     "wasm_examples/raster_tiles",
     "wasm_examples/vector_tiles",
     "wasm_examples/feature_layers",
     "wasm_examples/many_points",
     "wasm_examples/lambert",
+    "wasm_examples/with_egui",
 ]
 
 [workspace.package]
-version = "0.1.0-alpha.0"
 authors = ["Maxim Gritsenko <maxim@gritsenko.biz>"]
+categories = ["science::geo"]
 edition = "2021"
-repository = "https://github.com/Maximkaaa/galileo"
-license = "MIT OR Apache-2.0"
 keywords = ["gis", "map", "rendering"]
-categories = ["science::geo"]
\ No newline at end of file
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/Maximkaaa/galileo"
+version = "0.1.0-alpha.0"
```

Comment:
Apologies for the different file reformats, this is my editor's auto-format feature (VSCode, Even Better Toml for .toml and Prettier for .md).

Reply:
np